web/.env.example

```
@@ -0,0 +1,10 @@
+# Secrets — copy to web/.env and fill in locally; deploy fills /etc/combsense-web/env
+DJANGO_SECRET_KEY=change-me-to-a-50-char-random-string
+DJANGO_DEBUG=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DJANGO_DEBUG key should go before the DJANGO_SECRET_KEY key

(UnorderedKey)